### PR TITLE
Pure Swift library for iOS to create VCs

### PR DESCRIPTION
**Pull request by gauravping**
Pure Swift package for creating Verifiable Credentials (VCs) in multiple formats. This library helps in creation and verification of VCs formatted as SD-JWT (https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/) and VC-JWT (https://identity.foundation/jwt-vc-presentation-profile/)

### CODE REVIEW

**Comment by lukasjhan** [comment]
Hi @gauravping,

This link: (https://identity.foundation/jwt-vc-presentation-profile/) seems to be broken. If you look https://github.com/decentralized-identity/jwt-vc-presentation-profile, the link itself seems to be fine, but has the page disappeared?

File: proposal-template.md
Comment (tkuhrt) [inline proposal-template.md:1]: Please do not delete the template

**Comment by gauravping** [comment]
@lukasjhan - Thanks for pointing that out. I must have referenced that document no more than a week ago. Let me track that down and update this document and PR accordingly.

File: proposal-template.md
Comment (gauravping) [reply proposal-template.md:1]: Undeleted the proposal-template.md

**Comment by skounis** [comment]
@gauravping It looks like the new file is created in the repository's root folder. Can we please move it under the `projects` folder? 

https://github.com/openwallet-foundation/project-proposals/tree/main/projects

**Comment by skounis** [comment]
@gauravping The commits are incorrectly signed. Could you please check point "4. Commit your changes with proper sign-off" and section Fixing DCO on Existing Repositories" in the readme file?

* https://github.com/openwallet-foundation/project-proposals?tab=readme-ov-file

**Comment by gauravping** [comment]
@skounis - First time contributor mistakes. Thanks for all the references. Would you recommend restarting the PR with a fresh fork?

**Comment by tkuhrt** [comment]
> @skounis - First time contributor mistakes. Thanks for all the references. Would you recommend restarting the PR with a fresh fork?

You can either do that or you can follow the instructions under the [DCO details - Rebase section](https://github.com/openwallet-foundation/project-proposals/pull/27/checks?check_run_id=20532620167) to fix the existing commits with the DCO signoff.

**Comment by gauravping** [comment]
> Hi @gauravping,
> 
> This link: (https://identity.foundation/jwt-vc-presentation-profile/) seems to be broken. If you look https://github.com/decentralized-identity/jwt-vc-presentation-profile, the link itself seems to be fine, but has the page disappeared?

@lukasjhan - The link has been restored. There was some update on the DIF website which caused that link to disappear.

**Comment by tkuhrt** [comment]
Unanimously approved by the TAC at the [January 24, 2024 meeting](https://tac.openwallet.foundation/meetings/2024/2024-01-24/)

**Pull request merged**